css-animations-2/Overview.bs

```diff
@@ -962,14 +962,67 @@ The values of 'animation-trigger-exit-range-start' have the following meaning:
                         it defaults to 100%.
 </dl>
 
+## The 'trigger-name' property ## {#trigger-name}
+
+The 'trigger-name' property causes an [=event-based animation trigger=] to be created
+for any animation whose target element has an 'animation-trigger-event' property
+with matching <<trigger-name>>.
+When an {{Event}} is dispatched with the element having the matching 'trigger-name' property
+as {{Event/target}},
+and with {{Event/type}} matching the <<trigger-event-type>> of the
+'animation-trigger-event' property on the animation target, the animation's
+playback is modified according to its
+'animation-trigger-behavior' property, as described for
+[=event-based animation triggers=].
+
+<pre class='propdef'>
+Name: trigger-name
+Value: none | [ <<trigger-name>> ]#
```

Comment:
If this property has no other sub-properties, why does  it need to be a list?
IIUC this should simply be `none | <<trigger-name>>`